### DISCUSSION TITLE
咋不弄个官方的交流群呢?

### DISCUSSION THREAD

**Discussion by chenc041** [DouyinFE/semi-design#613] [General]
如题....

**Comment by pointhalo** [comment]
目前有微信群、飞书群的~（精力有限，所以像Discord之类的暂时没有考虑）

https://bytedance.feishu.cn/docs/doccnw93Dujm3UCkHRDTMTm1qwe

**Reply by chenc041** [reply]
好吧 我觉得可以在比较显眼的地方 放个二维码啥的
因为在我发这个之前 我一直以为没有官方的交流群.....

**Reply by pointhalo** [reply]
这个文档link放在了readme的Join User Group里，不是图片，所以相对来说可能没有那么显眼😂

**Reply by chenc041** [reply]
是的 而且还靠近底部....
个人建议可以弄个 issue 钉在那边, 这样比较显眼....
